src/buildercore/terraform.py

```diff
@@ -92,6 +92,11 @@ def render_fastly(context):
     if not context['fastly']:
         return {}
 
+    def _guess_shield(region):
+        return {
+            'us-east-1': 'iad-va-us',
```

Comment:
Obviously an incomplete list. Could be made a setting, but not sure why you wouldn't want it closest to the origin.

Reply:
Can be expanded if needed, clear enough